workloads/server.hbs.md

```diff
@@ -0,0 +1,98 @@
+# Using Server workloads
+
+## <a id="overview"></a>Overview
+
+The `tcp` workload type allows you to deploy traditional network applications on
+Tanzu Application Platform.
+Using an application workload specification, you can build and deploy application
+source code to a manually-scaled Kubernetes deployment which exposes an in-cluster Service endpoint.
+If required, you can use environment-specific LoadBalancer Services or Ingress resources to
+expose these applications outside the cluster.
+
+The `tcp` workload is a good match for traditional applications, including HTTP applications,
+that are implemented as follows:
+
+* Store state locally
+* Run background tasks outside of requests
+* Provide multiple network ports or non-HTTP protocols
+* Are not a good match for the `web` workload type
+
+Applications using the `tcp` workload type have the following features:
+
+* Do not natively autoscale, but can be used with the Kubernetes Horizontal Pod Autoscaler
+* By default are exposed only within the cluster using a `ClusterIP` Service
```

Comment:
Is there documentation we can link that shows the user how to configure a `ClusterIP` Service?

Reply:
The workload will configure the `ClusterIP` service itself; the user doesn't need to do anything but use the workload type.